RELEASE-NOTES.txt

```diff
@@ -2,6 +2,7 @@
 
 13.2
 -----
+- [**] Payments: UK merchants can collect In-Person Payments [https://github.com/woocommerce/woocommerce-ios/pull/9415]
```

Comment:
To avoid confusion between the store and the merchant being different, how about changing the release note to something like?
```suggestion
- [**] Payments: UK-based stores can collect In-Person Payments [https://github.com/woocommerce/woocommerce-ios/pull/9415]
```